app/build.gradle

```diff
@@ -78,16 +78,17 @@ play {
 }
 
 dependencies {
-    final PLAY_SERVICES_VERSION = '9.2.0'
-    final SUPPORT_LIBRARY_VERSION = '24.0.0'
+    final PLAY_SERVICES_VERSION = '9.6.1'
+    final SUPPORT_LIBRARY_VERSION = '24.2.1'
     final RETROFIT_VERSION = '2.1.0'
     final DAGGER_VERSION = '2.5'
     final DEXMAKER_VERSION = '1.4'
     final HAMCREST_VERSION = '1.3'
     final ESPRESSO_VERSION = '2.2.1'
     final RUNNER_VERSION = '0.4'
-    final BUTTERKNIFE_VERSION = '8.1.0'
-    final AUTO_VALUE_VERSION = '1.2'
+    final BUTTERKNIFE_VERSION = '8.4.0'
+    final AUTO_VALUE_VERSION = '1.3'
+    final AUTO_GSON_VALUE_VERSION = '0.4.2'
```

Comment:
I think this might be clearer if named consistently with the extension 'auto-value-gson', so `final AUTO_VALUE_GSON_VERSION = '0.4.2'`